src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -66,6 +66,9 @@
   /** The default port of the service. */
   private static final int DEFAULT_SERVICE_PORT = 443;
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
@bjwatson Your conceptualization of `gapic/` makes it completely useless for Java, because the generated code forms part of `google-cloud` and thus the version of `gccl` === the version of the generated code. 

For NodeJS/Ruby, where is this version used?